tests/test_flask_helpers/app_utils.py

```diff
@@ -67,7 +67,7 @@ class Config:
 )
 
 
-def test_action_enumeration(
+def mock_action_enumeration_func(
```

Comment:
I renamed these functions because they were named with `test_` which caused pytest to think they were tests to be run when in fact they are utility callables for other tests to supply to the action provider blueprint as action provider view functions.

Example failure: https://github.com/globus/action-provider-tools/actions/runs/6510610219/job/17684687247?pr=105